src/chapters/control-flow-and-collections/dictionary.rst

```diff
@@ -0,0 +1,250 @@
+.. index:: ! Dictionary, ! map
```

Comment:
we might want to steer away from map? im not sure that its a generic term so it might just be more appropriate to replace it with terms like dictionary and/or pair for a verb

Reply:
Done!